tunedev/Banka-react
# #167122085 Render user client pages

#### What does this PR do?
Adds articles page

#### Description of Task to be completed?
> - render sign-up page
> - render sign-in page
> - render summary page
> - render transaction history page
> - render landing page
> - render create account page

#### How should this be manually tested?
> - Clone this branch
> - Then run `npm install`
> - Then run `npm run dev' 
> - the app should auto display on your default browser

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#167122085](https://www.pivotaltracker.com/story/show/167122085)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
